test/Microsoft.Dnx.Testing.Framework/ExecResult.cs

```diff
@@ -15,7 +16,13 @@ public ExecResult EnsureSuccess()
         {
             if (ExitCode != 0)
             {
-                throw new InvalidOperationException($"Exit code was {ExitCode}");
+                var messageBuilder = new StringBuilder();
+                messageBuilder.AppendLine($"Exit code was {ExitCode}");
```

Comment:
nit: generally we use string interpolation in the style of https://github.com/aspnet/dnx/blob/dev/src/Microsoft.Dnx.ApplicationHost/DefaultHost.cs#L73 since it avoids creating a new `StringBuilder` object.